gno.land/pkg/sdk/vm/keeper.go

```diff
@@ -64,8 +65,9 @@ type VMKeeper struct {
 	baseKey store.StoreKey
 	iavlKey store.StoreKey
 	acck    auth.AccountKeeper
-	bank    bank.BankKeeper
+	bank    *bank.BankKeeper
```

Comment:
why did this have to change? there's a change in security (keepers are ideally immutable, though if we have mutable fields maybe this is OK, but managing mutable fields in keepers is tricky. 

See https://github.com/gnolang/gno/pull/3176/files#r1908750448

Reply:
Reverted. We no longer use mutable states in the bank keeper.